specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-11-02-privatepreview/AlertsManagement.json

```diff
@@ -0,0 +1,2062 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-05-05-preview",
```

Comment:
is this expected? should the version be `2018-11-02-preview`?

Reply:
fixed in latest commit. Also, this preview version is just for the new resource type. The old resource types continue with the old version. So, should the old resource type definitions be removed from this .json file?